src/runtime/matlab.cpp

```diff
@@ -0,0 +1,376 @@
+#include "HalideRuntime.h"
+
+#define INLINE inline __attribute__((always_inline))
+
+// This cannot be a function because the result goes out of scope.
+#define alloca __builtin_alloca
```

Comment:
IMO this indirection is not worthwhile, and you should just call __builtin_alloca below